docker/web/development/php.ini

```diff
@@ -1,6 +1,6 @@
 date.timezone = Europe/Amsterdam
 
-memory_limit = 256M
+memory_limit = -1
```

Comment:
I know this is for the development image, but do we really need this to be "unlimited"?

Reply:
These settings are taken from the PhpUnit documentation. These are the advised settings.